std/Date.hx

```diff
@@ -122,56 +121,52 @@ extern class Date
 		The first two formats are expressed in local time, the third in UTC
 		Epoch.
 	**/
-	static function fromString( s : String ) : Date;
-
-
-#if flash
-	private static function __init__() : Void untyped {
-		var d : Dynamic = Date;
-		d.now = function() {
-			return __new__(Date);
-		};
-		d.fromTime = function(t){
-			var d : Date = __new__(Date);
-			d.setTime(t);
-			return d;
-		};
-		d.fromString = function(s : String) {
-			switch( s.length ) {
-			case 8: // hh:mm:ss
-				var k = s.split(":");
-				var d : Date = __new__(Date);
-				d.setTime(0);
-				d.setUTCHours(k[0]);
-				d.setUTCMinutes(k[1]);
-				d.setUTCSeconds(k[2]);
+	static function fromString(s:String):Date;
+
+	#if flash
+	private static function __init__():Void
+		untyped {
```

Comment:
same here, untyped should be kept on declaration line

Reply:
Agree, there's an open issue for this: https://github.com/HaxeCheckstyle/haxe-formatter/issues/362